gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/CourseGradeOverridePanel.java

```diff
@@ -251,17 +259,54 @@ private String getGradeFromNumber(String newGrade, Map<String, Double> schema, L
 					if (maxValue.compareTo(tempValue) < 0) maxValue=tempValue;
 					if ((dValue.compareTo(tempValue) > 0 ) && (tempValue.compareTo(currentGradeValue) >= 0 )) {
 						currentGradeValue = tempValue;
-						newGrade=entry.getKey();
+						returnGrade=entry.getKey();
 					}
 				}
 				if (dValue < 0) throw new NumberFormatException("Grade cannot be lower than 0.");
 				if (dValue.compareTo(maxValue) > 0 && dValue > 100) throw new NumberFormatException("Grade exceeds the maximum number allowed in current scale.");
 			}
-			return newGrade;
+			return returnGrade;
 		}
 		catch (NumberFormatException e) {
 			throw new NumberFormatException("Grade is not a number, neither a scale value.");
 		}
 	}
 
+	/**
+	 *
+	 * @param gradeScale the scale value to convert
+	 * @param schema the current schema of Gradebook
+	 * @param currentUserLocale the locale to format the grade with the right decimal separator
+	 * @return the grade
+	 */
+	protected static String getNumberFromGrade(String gradeScale, Map<String, Double> schema, Locale currentUserLocale) {
```

Comment:
can we put these two methods into a utility or helper class so they can be tested and re-used?

Reply:
Where should I put them? On https://github.com/sakaiproject/sakai/tree/master/kernel/kernel-util/src/main/java/org/sakaiproject/util?

Reply:
FormatHelper.java?

Reply:
Done. Should I change the name: TestCourseGradeOverridePanel.java? What name it should be? TestFormatHelper.java?